deploy/crds/crd-clusterissuers.yaml

```diff
@@ -59,6 +59,10 @@ spec:
                     - privateKeySecretRef
                     - server
                   properties:
+                    caBundle:
+                      description: Base64-encoded bundle of PEM CAs which can be used to validate the certificate chain presented by the ACME server. Mutually exclusive with SkipTLSVerify; prefer using CABundle to prevent various kinds of security vulnerabilities. If CABundle and SkipTLSVerify are unset, the system certificate bundle inside the container is used to validate the TLS connection.
```

Comment:
As discussed at standup, the value should be a concatenated series of PEM blocks, and I think the documentation should should say something like:

> A series of PEM encoded CA certificates....
https://pkg.go.dev/crypto/x509#CertPool.AppendCertsFromPEM

Or

> A CA certificate store in PEM format...
https://curl.se/docs/caextract.html

Reply:
> the value should be a concatenated series of PEM blocks

This is in direct contradiction with what was said when this comment was originally added (see [here](https://github.com/cert-manager/cert-manager/pull/911#discussion_r218837389))

I've also tested this locally with a Vault issuer. I downloaded [this single PEM certificate](https://letsencrypt.org/certs/isrg-root-x1-cross-signed.pem) and the validating webhook rejects that file when passed as a single PEM certificate string with:

> admission webhook "webhook.cert-manager.io" denied the request: illegal base64 data at input byte 0

If I run base64 on the same input certificate and then add it to the caBundle field, it works entirely as expected.

As such, I think my change is good, at least for Vault. Do you have any examples showing why it's _not_ correct to state this for Vault? Is it different for Venafi? Should I change it for the ACME caBundle I'm adding in this PR?